### PR TITLE
Implement FNN training agent

### PR DESCRIPTION
**Pull request by robilkot**
# Trainer agent is now working.

## Capabilities

<details><summary>Specification of FNN structure in kb
</summary>
<p>

image

</p>
</details> 

<details><summary>Specification of preproccessed csv-datasets in kb</summary>
<p>

image

</p>
</details> 

<details><summary>Specification of training parameters in kb</summary>
<p>

image
</p>

</details> 

Given above parameters, agent is able to build tensorflow.keras FNN according to provided structure and train it on given dataset. Input and output data shapes are identified automatically. Trained model is then stored in filesystem, as well as linked to FNN structure in database, allowing for future interpretation

<details><summary>Example output during model training:</summary>
<p>

FNN structure per-layer is shown, as well as training state on each epoch

image

</p>
</details> 

[MNIST Fashion dataset](https://www.kaggle.com/datasets/zalando-research/fashionmnist) is used as example.

## Using training agent

1. Formalize desired FNN structure by providing layers with their sizes and activation functions in ordered way
2. Provide dataset: specify filepath and labels column
3. Provide desired training parameters: include learning rate, batch size and number of epochs
4. Create edge from 'question_initiated' node to action node
5. Wait for appearence of link containing path to file with trained keras model

_Trained models are stored at kb\fnn_processing_module\trained_models_

### CODE REVIEW

**Comment by robilkot** [comment]
Moved docker changes to #35

**Review by KovalM — APPROVED** [review]
